### ISSUE TITLE
Firefox: Alt text for images on article pages not being declared before the caption beneath the image

### ISSUE THREAD

**Issue by alicerichmond** [wellcomecollection/wellcomecollection.org#6006]
**What**

The Alt text for the images within articles is being read out after the caption beneath the image which makes the page compositon confusing if you are using a screen reader. 

It is  not in the correct order. The screen reader should be reading vertically down the page so the image should be declared with the alt text and then the quote under it. 

This may be because there is what looks like an image/fig caption being used for a quote. If there is a fig caption then the alt text would usually be excluded as it would be read twice. So its possible it is due to composition rather than code.

**Where**

<img width="1118" alt="Screenshot 2021-01-21 at 16 16 56" src="https://user-images.githubusercontent.com/7315126/105379552-d77f4980-5c04-11eb-9e13-a101f284b692.png">

**Comment by davidpmccormick** [comment]
This does appear to be consistent behaviour in Firefox ([minimal reproducible example in Codepen](https://codepen.io/davidpmccormick/pen/rNZdqyQ)). I haven't been able to find an obvious way around it while continuing to use the `figure` and `figcaption` elements which feel highly semantically appropriate. An obvious alternative is to use `div`s, but I think this would be worse for other reasons.
